pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -61,7 +61,7 @@ var updateBackoff = wait.Backoff{
 	Jitter:   1.0,
 }
 
-var errCouldNotFindMCPSet = newForgetError(errors.New("could not find any MachineConfigPool set for KubeletConfig"))
+var errCouldNotFindMCPSet = errors.New("could not find any MachineConfigPool set for KubeletConfig")
```

Comment:
I didn’t take a stab in the BZ as to why this was originally a forgettable error but maybe you remember (non blocking ofc)

Reply:
I remember writing this code, I think it was just an oversight on my part.